src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1026,14 +1014,6 @@ void runtime::cpu::CPU_ExternalFunction::build()
     //nv_cwi is required only by some frontends
     //in which case they should run this pass(CPUWorkspaceInsertion) explicitly
     NodeVector nv_cwi;
-    pass_manager.register_pass<ngraph::pass::NopElimination>();
-    pass_manager.register_pass<runtime::cpu::pass::LSTMFusion>();
-    pass_manager.register_pass<runtime::cpu::pass::RNNFusion>();
-    pass_manager.register_pass<runtime::cpu::pass::ConcatInputs>();
-    pass_manager.register_pass<ngraph::pass::AlgebraicSimplification>();
-    pass_manager.register_pass<ngraph::pass::CommonSubexpressionElimination>();
-    pass_manager.register_pass<ngraph::pass::CoreFusion>();
-    pass_manager.register_pass<runtime::cpu::pass::CPUFusion>();
     pass_manager.register_pass<runtime::cpu::pass::CPUWorkspaceInsertion>(nv_cwi);
     pass_manager.register_pass<runtime::cpu::pass::CPUAssignment>(this);
     pass_manager.register_pass<runtime::cpu::pass::CPULayout>(this);
```

Comment:
Cant we move these passes to register_and_run_passes?

Reply:
You mean these three passes: CPUWorkspaceInsertion, CPUAssignment, and CPULayout? There are more passes after these three. compile() needs pass_manager. I tried to make pass_manager a class member, and it caused problem.